### ISSUE TITLE
allow mouse panes resize. 

### ISSUE THREAD

**Issue by Sam-su2060** [zellij-org/zellij#2485]
Hello, 
it's good to do the resize with Ctrl-n when I am using the keyboard. 
It would be also more practical when I am using the mouse for scrolling to do panes resize immediately by mouse ( like in tmux). 
without the need to go back to the keyboard.

**Comment by jaeheonji** [comment]
I'm not sure but I think it depends on #1679

**Comment by Drugoy** [comment]
Duplicate of #1262
